doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+# Mount a Repo to a Local Computer
```

Comment:
General note:

I think most of the information in this document is correct (see other comments for correctness improvement), but the sections should be reordered. 

Specifically, the default setting and use case for `pachctl mount` is mount all repos in read-only. We should start with that simple case and then talk about more complicated things afterwards, building up. I envision the sections to flow similar to:

Opening: pc mount lets you mount data in pach as a local directory. These mounts can be read-only when you're just looking to experiment or consume the data, or in read/write mode which lets you commit changes to the mounted data back into Pachyderm. 
1. Prereqs
2. Basic/default usage of pachctl mount (read-only)
3. How to mount only specific repos, branches, or commits (instead of everything)
4. Mounting data in read/write mode (use with caution see below)
5. Important note!! Avoiding merge conflicts.
6. A simple walkthrough of using writable mount with opencv. Maybe a separate doc/example and just link to it? I'm not sure which is better

Reply:
Happy to discuss in TOH if you want. Again, in general the content is good, just need to fix the order the concepts are introduced

Reply:
so I think that prerequisites should go after. In the beginning, we discuss conceptual things - what is this feature, what are the use cases, limitations, etc. Then, we move on to the prerequisites and how to actually use it.

As for the example. What are you specifically looking for? There is a section below how to mount a Pachyderm repo. What else do you want to see there?